packages/client/lib/commands/CLUSTER_SET-CONFIG-EPOCH.spec.ts

```diff
@@ -0,0 +1,23 @@
+import { strict as assert } from 'assert';
+import testUtils, { GLOBAL } from '../test-utils';
+import { transformArguments } from './CLUSTER_SET-CONFIG-EPOCH';
+
+describe('CLUSTER SET-CONFIG-EPOCH', () => {
+    it('transformArguments', () => {
+        assert.deepEqual(
+            transformArguments(0),
+            ['CLUSTER', 'SET-CONFIG-EPOCH', '0']
+        );
+    });
+
+    testUtils.testWithCluster('cluster.clusterSetConfigEpoch', async cluster => {
+        try {
+            assert.equal(
+                await cluster.getSlotMaster(0).client.clusterSetConfigEpoch(1),
+                'OK'
+            );
+        } catch (ReplyError) {
```

Comment:
?

Reply:
Didn't know how to handle the error: 
`ReplyError: ERR The user can assign a config epoch only when the node does not know any other node.`

Reply:
Delete this test?

Reply:
yup